taskcluster/ci/client/kind.yml

```diff
@@ -30,6 +30,16 @@ tasks:
         {{ yarn --frozen-lockfile || exit 99; }} &&
         yarn test &&
         yarn package-test
+  nodejs-client-test:
+    description: nodejs client tests
```

Comment:
Maybe use a different description vs the task defined above this?

Reply:
nicely spot! will rename